### ISSUE TITLE
Grupo de Telegram

### ISSUE THREAD

**Issue by vidimus** [Anxelog/bailemascarasbaronesa#1]
Hola, Ánxelo.

Gracias a ti por crear un escenario y compartirlo con la comunidad. En realidad yo me dedico principalmente a traducir escenarios, que no han sido pocos. Sobre lo que te ha pasado, sí, sé exactamente cuál ha sido el problema por lo que has explicado. Le pasó algo parecido a dos autores italianos en su día y por el mismo motivo, que ponían el juego el inglés a la hora de jugar pero en italiano a la hora de editar (o al revés). Estupendo que lo hayas arreglado. Ahora a ver qué pasa con el fallo que han encontrado en Valkyrie y que de momento impide que puedan subir tu escenario. Espero que se solucione pronto. Yo ya tengo tu escenario instalado para jugar desconectado. En cuanto lo pruebe te dejaré mis impresiones.

Espero que no te importe que te ponga este mensaje aquí, es para no desviar el tema en la página de Valkyrie.

Hay un grupo de Telegram donde se habla de los escenarios y del editor de Valkyrie, para Descent y, sobre todo, para Mansiones.
Te pongo el enlace por si te quieres pasar por ahí.

https://t.me/joinchat/AAAAAEC2pI4jl7aLCQdq2A

Por si lo no conoces, hay varias hojas de cálculo donde puedes ver la puntuación que se le da a los escenarios y os comentarios que dejan los jugadores.
Aquí tienes una. Si quieres más, dímelo y te paso el resto.
[https://docs.google.com/spreadsheets/d/1BXDqQ4lpS3pG5SvwlRhkNtvw-xbmTH1E6c72tXuvpF8/edit?usp=drivesdk](url)

**Comment by Anxelog** [comment]
Buen día, Vidimus,

No conocía esas hojas de cálculo y agradezco mucho este link que me envías porque imagino que en el futuro podré obtener de ahí algún feedback. No me envíes más porque, de momento, les haría poco caso. Este verano trabajaré ya en un escenario más serio a nivel argumental y estilístico y ahí sí calculo que me interesará más conocer la opinión del público. Otro tanto en relación al grupo de Telegram.

Hacéis un trabajo extraordinario que, desde mi punto de vista, es trabajo voluntario y debería tener algún tipo de recompensa pública en forma de desgravación fiscal o algo así.

No me resisto a comentarte que dá gusto escribirse con alguien que domina tan bien la expresión escrita. Soy profesor de apoyo y a veces acabo la jornada muy pesimista en relación al estado de la lengua. Que alguien emplee un nivel como el tuyo para un e-mail esencialmente técnico me reconforta.

Un abrazo, Vidimus.

**Comment by vidimus** [comment]
Hola, @Anxelog . Cuando se publique tu escenario te pasaré otra que hay donde puedes obtener información útil como diseñador. 
Te pongo este enlace a un tutorial de Fernando Esra, por si no conoces el vídeo. Creo que te puede ayudar con tu próximo escenario. 
Tutorial de Valkyrie en Español - El vídeo está dividido en 3 partes de forma que la primera es un tutorial sencillo donde enseño a usar el programa y al final hay ejemplos prácticos para quienes ya sepan usarlo. Cualquier duda preguntad aquí o a mi en privado:

https://www.youtube.com/watch?v=qeKg9p5WkQM

Sobre la forma de escribir, creo que hay que mantener las formas y ser considerado con el lector. Lo que debería ser normal.

Para cualquier cosa en la que te pueda ayudar, no dudes en ponerte en contacto conmigo por Github.

Un saludo.

**Comment by Anxelog** [comment]
@vidimus El videotutorial al que haces referencia fue la fuente de información principal que utilicé para familiarizarme con el funcionamiento del editor. Te iba a decir que si tenías el contacto de quien lo hizo me lo pasases para agradecer el trabajo al hombre. Pero deduzco de tus palabras que el autor eres tú. Pues una vez más, Fernando, muchas gracias por tu generoso trabajo. El vídeo es realmente práctico y es el mejor tutorial que encontré (manejo con facilidad inglés, portugués y español; y busqué en todos esos idiomas). Gracias a ti creo que llegarán muchos escenarios en el futuro.

Te confieso que me llamó la atención tu manera de hablar en el vídeo, porque por momentos me recordaba a Galicia, pero no acabé de llegar a una conclusión cierta. Si no te parece indiscreto y me quieres satisfacer la curiosidad, me quedaré más tranquilo. Yo soy gallego.

Un abrazo

**Comment by vidimus** [comment]
No, no soy yo. Fernando Esra es otra persona. Ya me gustaría a mí saber tanto de Valkyrie y explicarlo tan bien. Lo tengo en un grupo de Telegram y en el canal de Discord de Valkyrie. Pero si quieres le puedo pedir un correo para que contactes con él.
Por cierto, él es extremeño. En tu caso, por el nombre y que te salía la cita en gallego, pues lo tenía más que claro.

Un saludo.

**Comment by Anxelog** [comment]
Si me consigues ese correo, te lo agradezco

**Comment by Anxelog** [comment]
He visto tu crítica al escenario y te la agradezco mucho, Vidimus. Tomo nota  y revisaré el escenario en los próximos días.

**Comment by vidimus** [comment]
> He visto tu crítica al escenario y te la agradezco mucho, Vidimus. Tomo nota y revisaré el escenario en los próximos días.

Hola, Ánxelo. La verdad es que la historia tiene gancho, pero creo que va muy precipitada. Tal vez no deberías dejar ir a las habitaciones laterales antes de hablar con el policía, y así sería un poco más lógico el orden. En mi caso, fue explorar las habitaciones primero, y claro, luego hablar con el policía... 

Te pongo capturas de algunas cosas que vi. Son erratas y un par de botones que salen en inglés. Y lo de la ficha de interacción, que debería desaparecer al interaccionar con ella con éxito.

![Captura de pantalla (786)](https://user-images.githubusercontent.com/41538948/122688809-b86dfd80-d21e-11eb-964a-270bc1d3393d.png)

![Captura de pantalla (788)](https://user-images.githubusercontent.com/41538948/122688803-b6a43a00-d21e-11eb-91ff-a1e78d8ba6ab.png)
![Captura de pantalla (789)](https://user-images.githubusercontent.com/41538948/122688806-b73cd080-d21e-11eb-9583-cc6eb73c46fa.png)
![Captura de pantalla (785)](https://user-images.githubusercontent.com/41538948/122688807-b7d56700-d21e-11eb-9e67-7ae552783d45.png)

![Captura de pantalla (787)](https://user-images.githubusercontent.com/41538948/122688801-b5730d00-d21e-11eb-9007-36cc84ae6e00.png)


Por cierto, pásame una dirección de correo para darte la de Fernando Esra, que me ha dado su número de móvil y permiso para compartir contigo esos datos. Dice que contactes con él por correo o whatsapp.

**Comment by vidimus** [comment]
El del viejo diario, para que expliques que es usando el botón de objetos de la aplicación, que hay que pensar que un escenario lo puede jugar alguien que no haya jugado nunca antes en Valkyrie y no sepa dónde hay que mirar.

**Comment by Anxelog** [comment]
Buen día, @vidimus. He corregido muchas de las cosas que comentabas (todos esos pantallazos no han caído en saco roto) e incluso otras que he encontrado al revisar más a fondo el escenario, para que etenga una calidad decente. Pero es para lo que es: pasárselo bien entre 60 y 90 min. No he añadido complejidad al escenario porque estoy trabajando en el nuevo, que será un esfuerzo más serio y, consiguientemente, exigirá más tiempo de juego.

Mi email: anxelog@gmail.com

Muchas gracias por todo, Vidimus. Continuamos en contacto
